source/common/router/config_impl.h

```diff
@@ -20,8 +20,11 @@
 #include "common/router/config_utility.h"
 #include "common/router/header_formatter.h"
 #include "common/router/header_parser.h"
+#include "common/router/metadatamatchcriteria_impl.h"
 #include "common/router/router_ratelimit.h"
 
+#include "extensions/filters/network/tcp_proxy/tcp_proxy.h"
+
```

Comment:
Shouldn’t the core code be completely independent of extensions ? ie one should be able to compile out extensions right?

Reply:
#3281